module-2/backend/week_two.md

```diff
@@ -5,19 +5,46 @@ Fork this respository. Answer the questions to the best of your ability. Try to
 Note: When you're done, submit a PR. 
 
 1. At a high level, what is ActiveRecord? What does it do/allow you to do?
+    It is ORM - Object Relational Mapping tool and allows us to create, read, update and delete records from our database and much much more. 
+    
 2. What is a migration?
+    A migration is a set of instructions to build a database table. 
+    
 3. How does a table relate to a model?
+    Every line in the table is an instance of a model. 
+
 4. What kind of methods are `belongs_to`, and `has_many`? (i.e. class or instance) Give an example.
+    They are instance methods, I think. In case of cars/dealerships, every instance of dealerships has many cars, whereas every instance of cars belongs to a dealership. 
```

Comment:
Good example! I think the point we were making with the class/instance distinction is that you'll need a specific instance of a Car to in order to call `.dealership` on it. E.g.:

```ruby
blue_pickup = Car.find(1) # finds the specific instance of a car with an id of 1.
mikes_trucks = blue_pickup.dealership # finds the dealership associated with that specific truck.
```